src/app/+dashboard/components/challenges/challenges.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+  <div class="header">
+    <p>CHALLENGES</p>
+    <button>Show more</button>
+  </div>
+
+  <div class="ch">
+    <div class="ach">
```

Comment:
Same here. It is not clear for what we use .ch and .ach classes. Please provide full names

Reply:
fixed